frontend/src/pages/org/browser-profiles-detail.ts

```diff
@@ -189,14 +174,42 @@ export class BrowserProfilesDetail extends TailwindElement {
                 ></sl-format-date>`
               : nothing}
           </btrix-desc-list-item>
+          ${
+            // NOTE older profiles may not have "modified/created by" data
```

Comment:
Should be fixed by the migration to be merged shortly but good to call this out for now